tests/configTest.php

```diff
@@ -51,7 +51,7 @@ function testConfigExportImportStatus() {
     $this->assertContains('unish', $page->front, 'Config was successfully imported.');
 
     // Test status of identical configuration.
-    $this->drush('config:status', array(), $options);
+    $this->drush('config:status --format=list', array(), $options);
```

Comment:
Instead, do $options + ['format' => 'list]

Reply:
FYI you can run test with `./unish.clean.php --filter testConfigExportImportStatus`